pkg/model/iam/iam_builder.go

```diff
@@ -878,25 +875,6 @@ func AddCCMPermissions(p *Policy, cloudRoutes bool) {
 				},
 			},
 		},
-		&Statement{
-			Effect: StatementEffectAllow,
-			Action: stringorslice.Of(
-				"elasticloadbalancing:CreateLoadBalancer",
-				"elasticloadbalancing:CreateLoadBalancerPolicy",
-				"elasticloadbalancing:CreateLoadBalancerListeners",
-				"ec2:CreateSecurityGroup",
-				"ec2:CreateVolume",
-				"elasticloadbalancing:CreateListener",
-				"elasticloadbalancing:CreateTargetGroup",
-			),
-			Resource: resource,
-
-			Condition: Condition{
-				"StringEquals": map[string]string{
-					"aws:RequestTag/KubernetesCluster": p.clusterName,
```

Comment:
The plan was to put all the create-like statements into a separate set and apply this condition. Removing the tag-on-create condition is perhaps less than ideal.